### PR TITLE
Update the dependency on the telemetry sdk 

### PR DESCRIPTION
**Pull request by jkwatson**
and add a secondary user-agent to the reporters.

### CODE REVIEW

File: src/main/scala/kamon/newrelic/spans/NewRelicSpanConverter.scala [hunk 1]
```diff
@@ -19,9 +19,9 @@ import kamon.util.Clock
 object NewRelicSpanConverter {
 
   /**
-   * Converts a Kamon Span$Finished instance into a New Relic Span
+   * Converts a Kamon Span.Finished instance into a New Relic Span
    *
-   * @param kamonSpan Kamon Span$Finished
+   * @param kamonSpan Kamon Span.Finished
```
Comment (breedx-nr) [inline src/main/scala/kamon/newrelic/spans/NewRelicSpanConverter.scala]: rad

Comment (jkwatson) [reply src/main/scala/kamon/newrelic/spans/NewRelicSpanConverter.scala]: sbt was yelping about it, so I fixed er up

**Comment by jkwatson** [comment]
If I were better at scala, I'd figure out where to put that common code. I can try to do that, if you think it would be a good change.

**Comment by breedx-nr** [comment]
I do think it would be a good change, but feel free to attack that in a followup PR.

**Comment by jkwatson** [comment]
ok, pushed something that does that...I'm 100% sure it's not scala idiomatic

File: src/main/scala/kamon/newrelic/LibraryVersion.scala [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import java.util.Optional
+
+object LibraryVersion {
+  val version: String =
+    Optional.ofNullable(classOf[LibraryVersion].getPackage.getImplementationVersion)
+      .orElse("UnknownVersion");
+}
+
+private class LibraryVersion
```
Comment (jkwatson) [inline src/main/scala/kamon/newrelic/LibraryVersion.scala]: I couldn't figure out how to get the class of the object, so I added this little dingle

Comment (breedx-nr) [reply src/main/scala/kamon/newrelic/LibraryVersion.scala]: Decent hack!

**Pull request merged**